great-idea/css/index.css

```diff
@@ -63,4 +63,260 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+/*desktop.jpg CSS code starts here*/
+.container{
+	width: 1000px;
+	margin: 0 auto;
+	display: flex;
+	flex-direction: column;
+}
+/*navigation */
```

Comment:
The separation of concerns and commenting that you are starting to build into your workflow such as an area for all of the nav elements, all of the header elements, etc is a really good practice to get into - Keep this up!